torchbox/torchbox.com
# Call to Action BE

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/views/6453316)

### Description of Changes Made
- Instantiated a CallToActionBlock
- This features three required fields; text, button_text and button_link
- The text and button_text have been limited to 255 and 55 chars respectively
- The button_link is a StreamBlock from which the user selects one of either an email, internal or external link
- I've created a basic template and yaml for the [complementary FE ticket](https://torchbox.monday.com/boards/1192293412/views/6453316?term=cta%20bloc&termColumns=XQAAAAJ6AAAAAAAAAABBKoI4ORlg8EVEDgl7wmWLVkOvg3GD4CMkswOCBFXe8hXiaSOX1WW-MVcUuC4pTt-m-tU6R4VLDl_WnsJlcQYQCpVujJ3wNbQW_zi_S_lbVF_t5WL151EikQuIFE89wFnJ4jv__-OYcAA)
(I've spoken to Chris about where to leave the template for handing over to him so have not done much with the FE)

### How to Test
- You should be able to add a Call to Action block to any page
- On this, you must set text, button_text and a button_link 
- The button_link can be either an email, internal or external link 
- These should render very simplistically in the browser as some text and a clickable link.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
